xml/System.Security.Claims/Claim.xml

```diff
@@ -475,9 +484,9 @@ if (null != principal)
         <ReturnType>System.Byte[]</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>Returns <see cref="T:System.Byte" />.</value>
-        <remarks>To be added.</remarks>
+        <summary>Contains any additional data provided by a derived type. Typically set when calling <see cref="M:System.Security.Claims.ClaimsIdentity.WriteTo(System.IO.BinaryWriter,System.Byte[])" />.</summary>
+        <remarks></remarks>
+        <value>A <see cref="T:System.Byte" /> value.</value>
```

Comment:
let's try to find out what the return value represents. The type is already automatically inserted by reflection.